scripts/build.sh

```diff
@@ -16,7 +16,7 @@ DISTDIR=dist
   mkdir -p $DISTDIR
   rm -rf $DISTDIR/*
 
- for d in opentelemetry-api/ opentelemetry-sdk/ opentelemetry-instrumentation/ opentelemetry-proto/ opentelemetry-distro/ exporter/*/ shim/*/ propagator/*/; do
+ for d in opentelemetry-api/ opentelemetry-sdk/ opentelemetry-proto/ exporter/opentelemetry-exporter-jaeger/ exporter/opentelemetry-exporter-otlp/ exporter/opentelemetry-exporter-zipkin/ propagator/*/; do
```

Comment:
any reason not to release a new version even for the packages that arent at 1.0.0? this would allow users to test the 1.0.0rc from all the packages

Reply:
If you mean why can't we release both `1.0.0rc1` AND `0.18b` at the same time, it would look strange since this release will be tagged as `1.0.0rc1`. I will be doing a release for `0.18b` shortly after this one that will depend on `1.0.0rc1` that will already be released.